modules/core/src/lib/attribute/attribute-manager.js

```diff
@@ -21,51 +21,19 @@
 /* eslint-disable guard-for-in */
 import Attribute from './attribute';
 import log from '../../utils/log';
+import debug from '../../debug';
+const debugLog = debug.log;
 
 import AttributeTransitionManager from './attribute-transition-manager';
 
-const LOG_START_END_PRIORITY = 2;
-const LOG_DETAIL_PRIORITY = 3;
-
-// Default loggers
-const logFunctions = {
-  savedMessages: [],
-  timeStart: null,
-  onUpdateStart: () => {
-    logFunctions.savedMessages.length = 0;
-    logFunctions.timeStart = Date.now();
-  },
-  onUpdate: message => {
-    logFunctions.savedMessages.push(message);
-  },
-  onUpdateEnd: message => {
-    const timeMs = Math.round(Date.now() - logFunctions.timeStart);
-    log.group(LOG_START_END_PRIORITY, `${message} in ${timeMs}ms`, {
-      collapsed: true
-    })();
-    for (const updateMessage of logFunctions.savedMessages) {
-      log.log(LOG_DETAIL_PRIORITY, updateMessage)();
-    }
-    log.groupEnd(LOG_START_END_PRIORITY)();
-  }
-};
+const EVENT_INVALIDATE = 'attributeManager.invalidate';
```

Comment:
Nit: Naming, `EVENT_...` sounds a bit asynchronous or user driven. Maybe `TRACE_...` ?

Reply:
Longer term: A question is whether there should be a central registry of these constants/enums (rather than having them defined locally in different files and relying on strings matching.

As a second step that could potentially be imported by apps to do custom things. I am thinking that could make sense if we were to expand on this to build e.g. the seer integration as a plugin.